processor/k8sattributesprocessor/e2e_test.go

```diff
@@ -47,7 +48,6 @@ func newExpectedValue(mode int, value string) *expectedValue {
 		value: value,
 	}
 }
```

Comment:
let's bring the line break back
```suggestion
}

```